RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.cc

```diff
@@ -54,16 +71,16 @@ HGCalRecHitWorkerSimple::run( const edm::Event & evt,
                               HGCRecHitCollection & result ) {
   DetId detid=uncalibRH.id();  
   uint32_t recoFlag = 0;
-  const std::vector<double>* fCPerMIP = nullptr;
+  //const std::vector<double>* fCPerMIP = nullptr;
     
   switch( detid.subdetId() ) {
   case HGCEE:
     rechitMaker_->setADCToGeVConstant(float(hgceeUncalib2GeV_) );
-    fCPerMIP = &HGCEE_fCPerMIP_;
+    //fCPerMIP = &HGCEE_fCPerMIP_;
     break;
   case HGCHEF:
     rechitMaker_->setADCToGeVConstant(float(hgchefUncalib2GeV_) );
-    fCPerMIP = &HGCHEF_fCPerMIP_;
+    //fCPerMIP = &HGCHEF_fCPerMIP_;
```

Comment:
better remove than comment out, unless there's a good reason to keep.
Add a comment why still needed.